### PR TITLE
GQty Example

### PR DESCRIPTION
**Pull request by elitan**
closing: https://github.com/nhost/nhost/issues/1027

### CODE REVIEW

File: examples/react-gqty/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "gqty",
```
Comment (plmercereau) [inline examples/react-gqty/package.json:2]:
```suggestion
  "name": "@nhost-examples/react-gqty",
```
turborepo relies on consistent names

File: examples/react-gqty/src/gqty/index.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      query,
+      variables
+    }),
+    mode: 'cors',
+    ...fetchOptions
+  })
+
+  const json = await response.json()
+
+  console.log(json)
```
Comment (plmercereau) [inline examples/react-gqty/src/gqty/index.ts:39]: can be removed I think

File: examples/react-gqty/src/components/SignIn.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  }
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <div>
+          <input type="email" placeholder="email" onChange={(e) => setEmail(e.target.value)} />
+        </div>
+        <div>
+          <button type="submit">Sign In</button>
```
Comment (plmercereau) [inline examples/react-gqty/src/components/SignIn.tsx:23]: I would be more explicit:
```suggestion
          <button type="submit">Sign In with a magic link</button>
```

**Review by plmercereau — CHANGES_REQUESTED** [review]
Very nice.
I would be good to add a mutation example e.g. add a post:
- to show how it works
- It's not a big deal for users that are used to the Hasura console, but I had to open it, go the users table, copy the user id, go the posts table and insert a post for the user. A bit rough for newcomers

File: examples/react-gqty/src/gqty/index.ts (hunk 2 above)
Comment (elitan) [inline examples/react-gqty/src/gqty/index.ts:39]:
```suggestion
```

**Comment by elitan** [comment]
> Very nice. I would be good to add a mutation example e.g. add a post:
> 
> * to show how it works
> * It's not a big deal for users that are used to the Hasura console, but I had to open it, go the users table, copy the user id, go the posts table and insert a post for the user. A bit rough for newcomers

Yea good point. I'll polish this PR a bit.

**Comment by elitan** [comment]
@plmercereau What am I doing wrong with the failing check here?

**Comment by plmercereau** [comment]
> @plmercereau What am I doing wrong with the failing check here?

You should always pay the check after diner, papa

**Comment by plmercereau** [comment]
image
In the root:
- pnpm run clean:all?
- pnpm I
- pnpn build
- pnpm Dev

Then go to your fancy new Ephesus of an example, and run both be and Fe performances. If he cries again, that's probably because it you

**Comment by elitan** [comment]
closing in favor of https://github.com/nhost/nhost/pull/1080

**Pull request closed without merging**